extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -584,54 +637,89 @@ export class AmpAnalytics extends AMP.BaseElement {
 
     const resultPromises = [];
     for (let r = 0; r < requests.length; r++) {
-      const request = this.requests_[requests[r]];
-      resultPromises.push(this.handleRequestForEvent_(request, trigger, event));
+      const requestName = requests[r];
+      resultPromises.push(
+          this.handleRequestForEvent_(requestName, trigger, event));
     }
     return Promise.all(resultPromises);
   }
 
   /**
+   * Check request is v1 or v2
+   * @param {string} request
+   * @return {boolean}
+   */
+  isV2Request_(request) {
```

Comment:
I'd rather have them all the same type.  The only difference to me, whether or not a `batch` option is available.

Reply:
removed v2